cobbler.spec

```diff
@@ -36,7 +36,7 @@
 %define apache_user www-data
 %define apache_group www-data
 %define apache_log /var/log/apache2
-%define apache_webconfigdir /etc/apache2/conf.d
+%define apache_webconfigdir /etc/apache2/conf-available
```

Comment:
I originally had `conf-available` but had to switch to `conf.d` because the `setup.py` did not set this up correctly. The logic there for the destination for the apache configs needs to be fixed.

Reply:
Apparently Ubuntu only builds with errors and Debian doesn't build at all. Could you please have a look? https://build.opensuse.org/package/show/home:SchoolGuy/cobbler-test

Reply:
```
[  105s] /bin/cp: cannot stat '/home/abuild/debbuild/BUILDROOT/cobbler-1579167507.b385f57f-1.1.amd64/etc/apache2/conf-available/cobbler.conf': No such file or directory
```

Reply:
@Conan-Kudo Are you also building on Debian Buster? because when I do "it just works" :)
See #2238

Reply:
Now it works apparently.